.gitignore

```diff
@@ -28,3 +28,6 @@ vendor/ruby
 
 # Generated js bundles
 /app/assets/javascripts/generated/*
+
+# Rubymine/IntelliJ
+.idea
```

Comment:
Totally agree on global ignore, here is mine: https://gist.github.com/samnang/a00a7b541f58f8401d4b